hexa/workspaces/graphql/schema.graphql

```diff
@@ -176,6 +191,7 @@ extend type Mutation {
   inviteWorkspaceMember(input: InviteWorkspaceMemberInput!): InviteWorkspaceMemberResult! @loginRequired
   updateWorkspaceMember(input: UpdateWorkspaceMemberInput!): UpdateWorkspaceMemberResult! @loginRequired
   deleteWorkspaceMember(input: DeleteWorkspaceMemberInput!): DeleteWorkspaceMemberResult! @loginRequired
+  getWorkspaceToken(input: GetWorkspaceTokenInput!): GetWorkspaceTokenResult! @loginRequired
```

Comment:
Could we call this `generateWorkspaceToken`?

Reply:
Only the first call of the mutation will create the token. The next ones will retrieve the value already in DB. That's why I think "generate" is not the appropriate verb. WDYT?

Reply:
I renamed it `generateWorkspaceToken`...